homeassistant/components/mold_indicator/strings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "name": "[%key:common::config_flow::data::name%]",
+          "calibration_factor": "Calibration factor",
+          "indoor_humidity_sensor": "Indoor humidity sensor",
+          "indoor_temp_sensor": "Indoor temperature sensor",
+          "outdoor_temp_sensor": "Outdoor temperature sensor"
+        },
```

Comment:
Please add also a `data_description` dictionary, in particular this is needed for the `calibration_factor`.